src/shell.zig

```diff
@@ -152,10 +152,9 @@ pub fn open_section(shell: *Shell, name: []const u8) !Section {
 const Section = struct {
     ci: bool,
     name: []const u8,
-    start: stdx.Instant,
+    timer: std.time.Timer,
```

Comment:
I don't want to import vsr in shell, so I refactored this one "for the worse" :D Doesn't matter for shell, I think.